src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -100,7 +100,7 @@ export class ApplicationStatus extends React.Component {
         : `Deleting your ${appType}.`;
 
       return (
-        <div className="sip-application-status">
+        <div className="sip-application-status vads-u-margin-bottom--2 vads-u-margin-top--0">
```

Comment:
Should the margin-bottom be 1 instead of 2?

Reply:
I got .vads-u-margin-bottom--2: 16px / 1em from https://design.va.gov/utilities/margins

Reply:
I think 1em is 16px

Reply:
That's a bit confusing, but it looks like the end result is right:

![image](https://user-images.githubusercontent.com/2008881/146994654-74c48cbf-ae40-46b1-b3a8-05254428e57b.png)